webapp/src/main/java/org/opentestsystem/rdw/admin/service/StudentGroupBatchService.java

```diff
@@ -0,0 +1,34 @@
+package org.opentestsystem.rdw.admin.service;
+
+import org.opentestsystem.rdw.admin.model.StudentGroupBatch;
+import org.opentestsystem.rdw.admin.security.User;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * Service responsible for managing {@link StudentGroupBatch}s
+ */
+//TODO: adjust permissions appropriately
```

Comment:
Yes I will file Jira, we also need to change yml
permissionservice:
  component: Reporting
 And create a new component in permissions service